man/common/authorizations.md

```diff
@@ -1,4 +1,13 @@
-# Password Formatting
+# Authorization Formatting
+
+Authorization for use of an object in tpm2.0 can come in 3 different forms:
+1. Password
+2. HMAC
+3. Sessions
+
+**NOTE:** Authorizations when not specified default to the **EMPTY PASSWORD**.
```

Comment:
This sentence seems hard to read for me, maybe "When none specified, authorizations default to the **EMPTY PASSWORD**" or "Authorizations defaults to the **EMPTY PASSWORD** when not specified".

Reply:
"Authorizations default to the EMPTY PASSWORD when not specified"